### PR TITLE
feat: add logos to homepage

### PR DESCRIPTION
**Pull request by takatoy**
127 0 0 1_5000_index html

### CODE REVIEW

**Comment by takatoy** [comment]
Closes #10 after merged into `main`.
This PR merges into `dev`.

`dev` will be merged after the check of Miraikan.

File: templates/index.html
Comment (toshi835) [inline templates/index.html:6]: [nit] can i ask why css styles are defined in html file? not in 'css' directory?

**Review by toshi835 — APPROVED** [review]
Thank you, looks good! who is trying to get the approval from miraikan?

**Comment by takatoy** [comment]
Apparently, the website has to be checked before being published:
https://inlg2024organizers.slack.com/archives/C06QUJ54A56/p1711516089295079

File: templates/index.html
Comment (takatoy) [reply templates/index.html:6]: For component-specific scoped styles like these, I believe co-locating them with the component HTML is cleaner.

**Pull request merged**
